pubsub/unit_tests/test_topic.py

```diff
@@ -779,6 +779,77 @@ def test_context_mgr_failure(self):
         self.assertEqual(list(batch.messages), [MESSAGE1, MESSAGE2])
         self.assertEqual(getattr(api, '_topic_published', self), self)
 
+    def test_message_count_autocommit(self):
+        """Establish that if the batch is assigned to take a maximum
+        number of messages, that it commits when it reaches that maximum.
+        """
```

Comment:
:-1: on docstring in a unit test. It mucks with test runner output and isn't worth it. (But :+1: on trying to be a good communicator)